design/open/$PULLS.md

```diff
@@ -0,0 +1,108 @@
+# $PULLS Proposal
+
+This proposal tries to bridge between two proposals: [pull-request](pull-request.md) and [checks](https://github.com/treeverse/lakeFS/blob/280c5b741df5e1e051958b8adde826364c6df614/design/open/branch_checks.md)
+and provide long-running CI/CD processes while laying the foundation for future "pull-request" like feature.
+
+## How it will work
+
+Introduce a new entity called $PULLS at the repository level. The $PULLS entity is identified by a source branch and destination branch. 
+Each source-destination pair can have only a single `OPEN` $PULLS instance.
```

Comment:
What's the harm in starting without this limitation?

Reply:
The purpose was to save on resources, and prevent redundant execution of a possibly very long process (action runs)
I don't think implementing it is a big issue